File: 백준/Bronze/2884. 알람 시계/알람 시계.py
h, m = (map(int, input().split()))

if(h==0 and m > 45) :
    print(f'{h} {m-45}')
elif(h==0 and m < 45) :
    print(f'{23} {(60-(45 - m))}')  
elif(h == h and m < 45) :
    print(f'{h-1} {60-(45-m)}')    
else :
    print(f'{h} {m-45}')
       

File: 백준/Bronze/2884. 알람 시계/알람 시계.cs
using System;

public class ing
{
    public static void Main()
    {
        int H, M;
        string[] ming = Console.ReadLine().Split(" ");
      
        H = int.Parse(ming[0]);
        M = int.Parse(ming[1]);

        if (H == 0 && M < 45)
        {
            Console.WriteLine("23" + " " + (60 - (45 - M)));
        } else if (H == 0 && M > 45)
        {
            Console.WriteLine(H+" "+ (M-45));
        } else if ( H == H && M < 45)
        {
            Console.WriteLine((H-1)+" "+ (60 - (45 - M)));
        }
        else
        {
            Console.WriteLine(H+" "+(M-45));
        }   
    }
}